modules/planning/tasks/deciders/st_bounds_decider/st_obstacles_processor.cc

```diff
@@ -169,19 +211,24 @@ bool STObstaclesProcessor::GetSBoundsFromDecisions(
   available_obs_decisions->clear();
 
   // Gather any possible change in st-boundary situations.
+  ADEBUG << "There are " << obs_t_edges_.size() << " t-edges.";
   std::vector<ObsTEdge> new_t_edges;
   while (obs_t_edges_idx_ < static_cast<int>(obs_t_edges_.size()) &&
          std::get<1>(obs_t_edges_[obs_t_edges_idx_]) <= t) {
     if (std::get<0>(obs_t_edges_[obs_t_edges_idx_]) == 0 &&
         std::get<1>(obs_t_edges_[obs_t_edges_idx_]) == t)
       break;
```

Comment:
Please also fix this though it's not introduced in this pull request.

Reply:
fixed it.